src/sv-pipeline/scripts/format_gatk_vcf_for_svtk.py

```diff
@@ -94,14 +94,26 @@ def convert(record: pysam.VariantRecord,
     """
     svtype = record.info['SVTYPE']
     alleles = record.alleles
+    # Reset alleles
+    if len(alleles) != 2:
+        raise ValueError(f"Encountered non-biallelic site {record.id} with alleles {alleles}")
+    if svtype == 'BND':
+        # Ensure we aren't using breakend notation here, since it isn't supported in some modules
+        alleles = ('N', '<BND>')
+    else:
+        alleles = ('N', alleles[1])
     contig = record.contig
     new_record = vcf_out.new_record(contig=contig, start=record.start, stop=record.stop, alleles=alleles)
     new_record.id = record.id
     # copy INFO fields
     for key in record.info:
         if key not in remove_infos:
             new_record.info[key] = record.info[key]
-    # fix END, CHR2, SVLEN, STRANDS
+    # svtk generally expects all records to have CHR2 assigned
+    chr2 = record.info.get('CHR2', None)
+    if chr2 is None:
+        new_record.info['CHR2'] = contig
+    # fix END, SVLEN, STRANDS
```

Comment:
Would it not be more correct to check `new_record.info` than `record.info`?

You can also write the check like the following, but both are functionally equivalent. 

```python
if "CHR2" not in record.info:
    new_record.info["CHR2"] = contig
```

Reply:
I did think about whether to use `new_record` or `record` for a moment, but I just thought this was a little safer in case the key-copying code above gets moved/deleted/changed.

I think we have to use the current logic as it's written. In VCFs, the field may exist but still be empty, i.e. if it's `CHR2=.`. In that case I believe `CHR2` in `record.info` will actually still return true but `record.info['CHR2']` will still return `None` in pysam.

Reply:
That makes sense; thanks for elaborating on it!